### DISCUSSION TITLE
Frunk & Trunk showing as unlocked

### DISCUSSION THREAD

**Discussion by abwino** [alandtse/tesla#238] [Q&A] [answered]
Hi friends,

Firstly - thank you @alandtse for making this integration possible! It is much appreciated. And thanks to all the contribuutors.

The frunk and trunk sensors on my Model 3 are reporting unlocked in HA even when they are locked. Has anyone else seen this? 
the integration is version 2.3.1

Many thanks,

Andrew.

**Comment by alexeiw123** [comment, marked as answer]
Yes I see this as well. If you toggle polling off then on they will correct themselves after a little while.

**Reply by abwino** [reply]
Hi, thanks very much for posting this. It worked fine for me, no more rogue lock notification! :)
